pkg/checker.go

```diff
@@ -0,0 +1,170 @@
+package pkg
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"golang.org/x/net/html"
+)
+
+const (
```

Comment:
please use https://github.com/fatih/color instead of maintaing ascii escape sequences in the project